### ISSUE TITLE
Doctrine orm second_level_cache default config are never used

### ISSUE THREAD

**Issue by I70l0teN4ik** [symfony/symfony#36881]
**Symfony version(s) affected**: appears on 4.4.8

**Description**  
Looks like default configs for second_level_cache are never applied. 
From what I understood such config should be applied to every region by default:
```
doctrine:
  orm:
    second_level_cache:
      enabled: true
      region_lifetime: 123
      region_lock_lifetime: 45
      region_cache_driver:
        type: service
        id: some_service
```
And basically that is never used. So default `Doctrine\ORM\Cache\RegionsConfiguration` `$defaultLifetime` value is always 3600 even if those configs are added.
And to define different lifetimes per region - `cache_driver` should be duplicated. e.g.:
```
doctrine:
  orm:
    second_level_cache:
      ...
     regions:
        default:
          lifetime: 369
          cache_driver:
            type: service
            id: some_service
        another_region:
          lifetime: 321
          cache_driver:
            type: service
            id: some_service
```

**Possible Solution**  
Values from `orm.region_*` confings should be used by default for all of the regions if they are defined.

**Additional context**  
I'm trying to use `doctrine second_level_cache` with `snc_redis` bundle . That bundle creates cach region per Entity by default. And that gives a bit better overview and better control of the cached values.
Basically if I could setup default values all I had to add would be the `ORM\Cache` annotation to Entity (relations) in most cases. Also I could control the `ttl` per Entity easily. And now lots of duplication must be added to make that work.

**Comment by xabbuh** [comment]
The `doctrine` config is not handled by the Symfony core, but by the [DoctrineBundle](https://github.com/doctrine/DoctrineBundle). Can you report it over there, please? Thank you!

**Comment by I70l0teN4ik** [comment]
Yes of course. Sorry for that. Just confused because of monorepo policy...
